hub/package-manager/winget/import.md

```diff
@@ -36,6 +36,7 @@ The options allow you to customize the import experience to meet your needs.
 | **--ignore-versions** |  Ignores versions specified in the JSON file and installs the latest available version. |
 | **--accept-package-agreements** | Used to accept the license agreement, and avoid the prompt. |
 | **--accept-source-agreements** | Used to accept the source license agreement, and avoid the prompt. |
+| **--verbose-logs** | Used to override the logging setting and create a verbose log. |
```

Comment:
-v?

Reply:
I don't think --verbose-logs has a short form since that would conflict with other commands that use -v like winget install which uses -v to specify --version.

Reply:
Ok, sounds good. Thanks for confirming.